### PR TITLE
All march/soloist concerts

### PR DESCRIPTION
**Pull request by jkuo168**
Added all historical march concerts

### CODE REVIEW

File: src/content/Concerts/00-01March.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Hall = Richardson Auditorium in Alexander Hall
+Generaltime = March 2001
+
+Section
+
+Overline = Program
+Work = Ludwig van Beethoven ~ *Leonore Overture No. 3*, op. 72a
+Work = Ralph Vaughan Williams ~ *Fantasia on a Theme by Thomas Tallis*
+Break = --- Intermission ---
+Work = W. A. Mozart ~ Bassoon Concerto in B-flat major, K.191/186e
```
Comment (ReillyBova) [inline src/content/Concerts/00-01March.md]: leave a space after k

File: src/content/Concerts/01-02March.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+Section
+
+Hall = Richardson Auditorium in Alexander Hall
+Datetime = March 9, 2002 19:30
+
+Section
+
+Overline = Program
+Work = Igor Stravinsky ~ *Symphonies of Wind Instruments*
+Work = W. A. Mozart ~ Violin Concerto No. 5 in A major, K. 219, *The Turkish* ~ Alexis Kende '05, violin
```
Comment (ReillyBova) [inline src/content/Concerts/01-02March.md]: Capitolize violin

File: src/content/Concerts/04-05March.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+# Ticket link (if applicable)
+tickets: ""
+# Livestream link (if applicable)
+stream: ""
+# Youtube link (if applicable)
+youtube: ""
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = The Magic Fire and the Farewell
```
Comment (ReillyBova) [inline src/content/Concerts/04-05March.md]: Ampersand

File: src/content/Concerts/04-05March.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+Datetime = March 5, 2005 19:30
+
+Section
+
+Overline = Program
+Work = W. A. Mozart ~ Symphony No. 39 in E-flat major, K. 543
+Work = Alfred Schnittke ~ Concerto for Viola and Orchestra ~ David Aaron Carpenter, Viola
+Break = --- Intermission ---
+Work = Richard Wagner ~ "Wotan's Farewell" from *Die Walküre*, WWV. 86B ~ Jonathan Prescott, Bass-baritone
+Work = Richard Wagner ~ "Magic Fire Music" from *Die Walküre*, WWV. 86B ~ Jonathan Prescott, Bass-baritone
```
Comment (ReillyBova) [inline src/content/Concerts/04-05March.md]: Include translation of the german

File: src/content/Concerts/05-06March.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+Section
+
+Hall = Richardson Auditorium in Alexander Hall
+Datetime = March 9, 2006 19:30
+
+Section
+
+Overline = Program
+Work = W. A. Mozart ~ Act I of *Die Walküre* (The Valkyrie), WWV. 86B
```
Comment (ReillyBova) [inline src/content/Concerts/05-06March.md]: MOZART!? 0.0

File: src/content/Concerts/00-01March.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2001-03-00"
+# Season
+season: "2000/01"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: ""
```
Comment (ReillyBova) [inline src/content/Concerts/00-01March.md]: add poster

Comment (ReillyBova) [reply src/content/Concerts/00-01March.md]: maybe "vaughn-williams"

File: src/content/Concerts/01-02March.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2002-03-09"
+# Season
+season: "2001/02"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: ""
```
Comment (ReillyBova) [inline src/content/Concerts/01-02March.md]: Add poster: "rimsky"

File: src/content/Concerts/02-03March.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2003-03-00"
+# Season
+season: "2002/03"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: ""
```
Comment (ReillyBova) [inline src/content/Concerts/02-03March.md]: add poster: "sibelius"

File: src/content/Concerts/03-04March.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2004-03-06"
+# Season
+season: "2003/04"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: ""
```
Comment (ReillyBova) [inline src/content/Concerts/03-04March.md]: add poster "stravinsky"

File: src/content/Concerts/04-05March.md [hunk 10]
```diff
@@ -0,0 +1,40 @@
...
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2005-03-05"
+# Season
+season: "2004/05"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: ""
```
Comment (ReillyBova) [inline src/content/Concerts/04-05March.md]: add poster: "mozart"

File: src/content/Concerts/05-06March.md [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2006-03-09"
+# Season
+season: "2005/06"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: ""
```
Comment (ReillyBova) [inline src/content/Concerts/05-06March.md]: add poster: wagner

File: src/content/Concerts/06-07Soloist.md [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2007-03-00"
+# Season
+season: "2006/07"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: ""
```
Comment (ReillyBova) [inline src/content/Concerts/06-07Soloist.md]: add poster "dvorak"

File: src/content/Concerts/06-07Soloist.md [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+# Ticket link (if applicable)
+tickets: ""
+# Livestream link (if applicable)
+stream: ""
+# Youtube link (if applicable)
+youtube: ""
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = The Violin and Cello
```
Comment (ReillyBova) [inline src/content/Concerts/06-07Soloist.md]: ampersand: &

File: src/content/Concerts/06-07Soloist.md [hunk 14]
```diff
@@ -0,0 +1,38 @@
...
+Subtitle = Michael Pratt, Conductor
+
+Section
+
+Hall = Richardson Auditorium in Alexander Hall
+Generaltime = March 2007
```
Comment (ReillyBova) [inline src/content/Concerts/06-07Soloist.md:31]: no date in the archives?

File: src/content/Concerts/07-08March.md [hunk 15]
```diff
@@ -0,0 +1,39 @@
...
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2008-03-07"
+# Season
+season: "2007/08"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: ""
```
Comment (ReillyBova) [inline src/content/Concerts/07-08March.md]: add poster: "ravel"

File: src/content/Concerts/07-08March.md [hunk 16]
```diff
@@ -0,0 +1,39 @@
...
+# Ticket link (if applicable)
+tickets: ""
+# Livestream link (if applicable)
+stream: ""
+# Youtube link (if applicable)
+youtube: ""
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = The Mother Goose Suite
```
Comment (ReillyBova) [inline src/content/Concerts/07-08March.md]: Maybe just: Mother Goose

File: src/content/Concerts/08-09March.md [hunk 17]
```diff
@@ -0,0 +1,42 @@
...
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2009-03-07"
+# Season
+season: "2008/09"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: ""
```
Comment (ReillyBova) [inline src/content/Concerts/08-09March.md]: add poster: gershwin

File: src/content/Concerts/09-10Soloist.md [hunk 18]
```diff
@@ -0,0 +1,41 @@
...
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2010-03-06"
+# Season
+season: "2009/10"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: ""
```
Comment (ReillyBova) [inline src/content/Concerts/09-10Soloist.md]: add poster: Beethoven

File: src/content/Concerts/09-10Soloist.md [hunk 19]
```diff
@@ -0,0 +1,41 @@
...
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = Beethoven Symphony No. 2
+Subtitle = Michael Pratt, Conductor
+
+Section
+
+Hall = Richardson Auditorium in Alexander Hall
+Datetime = March 6, 2010 19:30
```
Comment (ReillyBova) [inline src/content/Concerts/09-10Soloist.md:32]: only one?

File: src/content/Concerts/10-11Soloist.md [hunk 20]
```diff
@@ -0,0 +1,41 @@
...
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2011-03-05"
+# Season
+season: "2010/11"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: ""
```
Comment (ReillyBova) [inline src/content/Concerts/10-11Soloist.md]: add poster: sibelius

File: src/content/Concerts/10-11Soloist.md [hunk 21]
```diff
@@ -0,0 +1,41 @@
...
+# Ticket link (if applicable)
+tickets: ""
+# Livestream link (if applicable)
+stream: ""
+# Youtube link (if applicable)
+youtube: ""
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = Dancigers of the Electric Guitar
```
Comment (ReillyBova) [inline src/content/Concerts/10-11Soloist.md]: lol..... idk about including the last name. How about "Rocking Richardson"

File: src/content/Concerts/10-11Soloist.md [hunk 22]
```diff
@@ -0,0 +1,41 @@
...
+Hall = Richardson Auditorium in Alexander Hall
+Datetime = March 4, 2011 19:30
+Datetime = March 5, 2011 19:30
+
+
+Section
+
+Overline = Program
+Work = Yngwie Malmsteen ~ Concerto Suite for Electric Guitar and Orchestra ~ Mark Dancigers, Electric Guitar
+Work = W. A. Mozart ~ Flute Concerto No. 2 in D Major, K. 314/ 285d ~ Jessica Anastasio ’11, Flute
```
Comment (ReillyBova) [inline src/content/Concerts/10-11Soloist.md]: spacing on slash

File: src/content/Concerts/11-12Soloist.md [hunk 23]
```diff
@@ -0,0 +1,38 @@
...
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2012-03-09"
+# Season
+season: "2011/12"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: ""
```
Comment (ReillyBova) [inline src/content/Concerts/11-12Soloist.md]: poster: rachmaninoff

File: src/content/Concerts/11-12Soloist.md [hunk 24]
```diff
@@ -0,0 +1,38 @@
...
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = The Symphonic Dances
+Subtitle = Michael Pratt, Conductor
+
+Section
+
+Hall = Richardson Auditorium in Alexander Hall
+Datetime = March 9, 2012 19:30
```
Comment (ReillyBova) [inline src/content/Concerts/11-12Soloist.md:31]: only one?

File: src/content/Concerts/11-12Soloist.md [hunk 25]
```diff
@@ -0,0 +1,38 @@
...
+
+Section
+
+Overline = Program
+Work = Frédéric Chopin ~ Piano Concerto No. 1 in E minor, op. 11 ~ Darya Koltunyuk ’15, Piano
```
Comment (ReillyBova) [inline src/content/Concerts/11-12Soloist.md:36]: omg she works for Princeton now!

File: src/content/Concerts/11-12Soloist.md [hunk 26]
```diff
@@ -0,0 +1,38 @@
...
+# Ticket link (if applicable)
+tickets: ""
+# Livestream link (if applicable)
+stream: ""
+# Youtube link (if applicable)
+youtube: ""
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = The Symphonic Dances
```
Comment (ReillyBova) [inline src/content/Concerts/11-12Soloist.md]: I like: "Chopin & Rachmaninoff" better

File: src/content/Concerts/12-13Soloist.md [hunk 27]
```diff
@@ -0,0 +1,43 @@
...
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2013-03-09"
+# Season
+season: "2012/13"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: ""
```
Comment (ReillyBova) [inline src/content/Concerts/12-13Soloist.md]: "wagner"

Comment (ReillyBova) [reply src/content/Concerts/12-13Soloist.md]: poster ^^^

File: src/content/Concerts/12-13Soloist.md [hunk 28]
```diff
@@ -0,0 +1,43 @@
...
+Section
+
+Hall = Richardson Auditorium in Alexander Hall
+Datetime = March 8, 2013 19:30
+Datetime = March 9, 2013 19:30
+
+Section
+
+Overline = Program
+Work = J. S. Bach ~ Piano Concerto No. 5 in F minor, DWV. 1056 ~ Ruth Ochs, Guest Conductor ~ Paul von Autenreid ’16, Piano
```
Comment (ReillyBova) [inline src/content/Concerts/12-13Soloist.md]: Don't add guest conductor. See 2016/17 season

Comment (ReillyBova) [reply src/content/Concerts/12-13Soloist.md]: Also fix the order (conductor always goes second for concerti)

File: src/content/Concerts/13-14Soloist.md [hunk 29]
```diff
@@ -0,0 +1,40 @@
...
+# Ticket link (if applicable)
+tickets: ""
+# Livestream link (if applicable)
+stream: ""
+# Youtube link (if applicable)
+youtube: ""
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = Prauge and Scotland
```
Comment (ReillyBova) [inline src/content/Concerts/13-14Soloist.md]: Try: Prague & Scotland

File: src/content/Concerts/13-14Soloist.md [hunk 30]
```diff
@@ -0,0 +1,40 @@
...
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2014-03-00"
+# Season
+season: "2013/14"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: ""
```
Comment (ReillyBova) [inline src/content/Concerts/13-14Soloist.md]: "mozart"

File: src/content/Concerts/13-14Soloist.md [hunk 31]
```diff
@@ -0,0 +1,40 @@
...
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = Prauge and Scotland
+Subtitle = Michael Pratt, Conductor
+
+Section
+
+Hall = Richardson Auditorium in Alexander Hall
+Generaltime = March 2014
```
Comment (ReillyBova) [inline src/content/Concerts/13-14Soloist.md:31]: no date?

File: src/content/Concerts/14-15Soloist.md [hunk 32]
```diff
@@ -0,0 +1,40 @@
...
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2015-03-05"
+# Season
+season: "2014/15"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: ""
```
Comment (ReillyBova) [inline src/content/Concerts/14-15Soloist.md]: poster: "respighi"

File: src/content/Concerts/14-15Soloist.md [hunk 33]
```diff
@@ -0,0 +1,40 @@
...
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = Pines of Rome
+Subtitle = Michael Pratt, Conductor
+
+Section
+
+Hall = Richardson Auditorium in Alexander Hall
+Datetime = March 5, 2015 19:30
```
Comment (ReillyBova) [inline src/content/Concerts/14-15Soloist.md]: only one?

File: src/content/Concerts/14-15Soloist.md [hunk 34]
```diff
@@ -0,0 +1,40 @@
...
+
+Section
+
+Overline = Program
+Work = Nicola Porpora and George Frederick Handel ~ Arias by Nicola Porpora and George Frederick Handel ~ Aryeh Nussbaum Cohen ’15, Countertenor
```
Comment (ReillyBova) [inline src/content/Concerts/14-15Soloist.md]: try: Nicola Porpora / George Frederick Handel ~ Selected Arias ~ Aryeh Nussbaum Cohen ’15, Countertenor

File: src/content/Concerts/96-97March.md [hunk 35]
```diff
@@ -0,0 +1,39 @@
...
+# Ticket link (if applicable)
+tickets: ""
+# Livestream link (if applicable)
+stream: ""
+# Youtube link (if applicable)
+youtube: ""
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = The Stravinsky Concert
```
Comment (ReillyBova) [inline src/content/Concerts/96-97March.md]: A Stravinsky Concert*

File: src/content/Concerts/97-98March.md [hunk 36]
```diff
@@ -0,0 +1,40 @@
...
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = Beethoven Symphony No. 7
+Subtitle = Michael Pratt, Conductor
+
+Section
+
+Hall = Richardson Auditorium in Alexander Hall
+Generaltime = March 1998
```
Comment (ReillyBova) [inline src/content/Concerts/97-98March.md:31]: no specific date?

File: src/content/Concerts/10-11Soloist.md (hunk 22 above)
Comment (ReillyBova) [reply src/content/Concerts/10-11Soloist.md]: there should be no space: 314/285d

File: src/content/Concerts/98-99March.md [hunk 37]
```diff
@@ -0,0 +1,38 @@
...
+
+Section
+
+Hall = Richardson Auditorium in Alexander Hall
+Generaltime = March 1999
+
+Section
+
+Overline = Program
+Work = W. A. Mozart ~ Concerto for Two pianos in E-flat major, K. 365/316a
```
Comment (ReillyBova) [inline src/content/Concerts/98-99March.md]: Pianos should be uppercase

File: src/content/Concerts/99-00March.md [hunk 38]
```diff
@@ -0,0 +1,37 @@
...
+spotify: ""
+---
+Title = Midsummer Night's Dream on Stage
+Subtitle = Michael Pratt, Conductor
+Subtitle = Princeton Shakespeare Company
+
+Section
+
+Hall = Richardson Auditorium in Alexander Hall
+Generaltime = March 2000
```
Comment (ReillyBova) [inline src/content/Concerts/99-00March.md:32]: no specific date?

**Review by ReillyBova — COMMENTED** [review]
Great job! Left some comments.... mostly needs posters and a few typos

File: src/content/Concerts/00-01March.md (hunk 6 above)
Comment (jkuo168) [reply src/content/Concerts/00-01March.md]: wrote vaughn-williams

File: src/content/Concerts/09-10Soloist.md (hunk 18 above)
Comment (jkuo168) [reply src/content/Concerts/09-10Soloist.md]: lower case beethoven correct?

File: src/content/Concerts/12-13Soloist.md (hunk 28 above)
Comment (jkuo168) [reply src/content/Concerts/12-13Soloist.md]: tried and fixed it don't know if it's right. 16/17 season had no guest conductor only guest groups

File: src/content/Concerts/05-06March.md [hunk 39]
```diff
@@ -0,0 +1,36 @@
...
+# Concert Color theme (use hex form of sRGB to find)
+colorTheme: "#8AC44A"
+# Date YY-MM-DD (for sorting)
+date: "2006-03-09"
+# Season
+season: "2005/06"
+# Is current season?
+current: "false"
+# Poster filename (without extension)
+poster: "wager"
```
Comment (ReillyBova) [inline src/content/Concerts/05-06March.md]: wagner

File: src/content/Concerts/05-06March.md [hunk 40]
```diff
@@ -0,0 +1,36 @@
...
+# Ticket link (if applicable)
+tickets: ""
+# Livestream link (if applicable)
+stream: ""
+# Youtube link (if applicable)
+youtube: ""
+# Spotify link (if applicable)
+spotify: ""
+---
+Title = Die Walkure
```
Comment (ReillyBova) [inline src/content/Concerts/05-06March.md]: missing special character 'ü'

**Review by ReillyBova — CHANGES_REQUESTED** [review]
Left a couple comments

File: src/content/Concerts/11-12Soloist.md (hunk 24 above)
Comment (jkuo168) [reply src/content/Concerts/11-12Soloist.md:31]: oui

**Review by ReillyBova — APPROVED** [review]

**Pull request merged**